app/repositories/paper_repository.py

```
@@ -6,6 +6,8 @@
 
 from pymongo.database import Database
 from pymongo.collection import Collection
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionError as ESConnectionError
```

Flake8 (7.3.0)

[error] 'elasticsearch.exceptions.ConnectionError as ESConnectionError' imported but unused

(F401)